.github/workflows/publish.yml

```diff
@@ -80,6 +71,6 @@ jobs:
         run: |
           npm version --no-git-tag-version ${GITHUB_REF/refs\/tags\//}
```

Comment:
todo: this version update has not succeeded for me in my fork. something may not be working correctly here

Reply:
this should be fixed by switching to a mac build agent for publishing: https://github.com/chrisfromwork/BabylonReactNative/runs/1858031216?check_suite_focus=true

I was unable to confirm whether multiple packages in the packaging action task broke this version issue, but it seems to work/we'll only be able to test this with the correct permissions provided by the BabylonJS/BabylonReactNative repo.